installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oci-assisted-installer"]
+= Using the Assisted Installer to install a cluster on OCI
+include::_attributes/common-attributes.adoc[]
+:context: installing-oci-assisted-installer
+
+toc::[]
+
+In {product-title} {product-version}, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+
+// The Assisted Installer and OCI overview
+include::modules/installing-oci-about-assisted-installer.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.html#cluster-entitlements_installing-platform-agnostic[Internet access for {product-title}]
+* link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units]
+* xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.html#using-the-assisted-installer_installing-on-prem-assisted[Using the Assisted Installer]
+
+// Using the Assisted Installer to install a cluster on OCI
+include::modules/installing-using-assisted-installer-oci.adoc[leveloffset=+1]
+
+// Troubleshooting installation of a cluster on OCI
+include::modules/installing-troubleshooting-assisted-installer-oci.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#using-the-assisted-installer_installing-on-prem-assisted[Installing an on-premise cluster using the Assisted Installer]
+* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[Assisted Installer for {product-title}]
+//* link:https://access.redhat.com/node/7038262[Using the Agent-based Installer to install a cluster on OCI]
```

Comment:
Will comment in when PR is created for migrating the Agent-based Installer work from a KBase article to the `openshift-docs` repo.